### ISSUE TITLE
improve error reporting for http timeout during fin up?

### ISSUE THREAD

**Issue by joachim-n** [docksal/docksal#1167]
**Description**

I couldn't do `fin up` because my mac had port forwarding set up for port 80. Is there a way to make the error message indicate what the problem is?

**Steps to reproduce the issue:**

1. Set up port forwarding from 8888 to 80 (this was for using MAMP)
2. Do `fin up`

**Describe the results you received:**

```
UnixHTTPConnectionPool(host='localhost', port=None): Read timed out. (read timeout=60)
```

**Describe the results you expected:**

Would be more helpful if there was a message saying what the source of the problem is, assuming this is possible?

**Comment by lmakarov** [comment]
There is an [existing (old) issue](https://github.com/docksal/docksal/issues/176) and an [attempt to improve error handling](https://github.com/docksal/docksal/commit/d3744f10733f4ac6ffc7add33d1b2e4213f97235) for such cases. IIRC there were issues with checking port availability across all systems. I may try giving it another spin.

Closing as a dupe of #176

**Comment by joachim-n** [comment]
Thanks!

**Comment by achekulaev** [comment]
Frankly I did not understand the issue. If port 80 was unavailable when starting Docksal, then Docksal would not start at all giving you a big error about that. So it was available. I don't really understand on what stage have you had the error you had. Was the error in console or browser. And when.

**Comment by joachim-n** [comment]
I got the error in my terminal when I tried to do `fin up`.

I had port forwarding set up on OS X so that sites served with MAMP could be reached on port 80 rather than 8888. This was done with a LaunchDaemon that runs `pfctl`, following the instructions in the 'Run with port 80' section towards the end of this page: https://alan.ivey.dev/posts/2014/os-x-10.9-local-development-environment-apache-php-and-mysql-with-homebrew/

When I removed the LaunchDaemon and restarted my machine, `fin up` worked as expected with no error.
